src/services/upload.service.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import fileUpload from 'express-fileupload';
+import sendResult from '../utils/sendResult';
+import db from '../database/models/index';
+import cloudinary from '../config/clound-config';
+
+
+let success = 0;
+let failure = 0;
```

Comment:
Nice Job @Crispy-lang 
However, the values of `Global variables` are maintained in all life circle of an application.
it means you should have declared them inside a function scope.
Otherwise, it will cause some un-expected logic errors. (`because their current values will be maintained until the next HTTP request`)

Reply:
thanks this feedback!